python-scripts/compare_localization_btw_env.py

```
@@ -0,0 +1,143 @@
+    for code in source_localization:
+        if (
+            code not in target_localization
+            and code["code"] not in NEGLECT_CODES
+            and code["module"] not in NEGLECT_MODULES
+        ):
+            missing_codes.append((code["code"], code["message"], code["module"]))
+
+    return missing_codes
+
+
+def save_missing_localizations(missing_codes, locale):
```

Pylint (3.3.7)

[warning] Redefining name 'locale' from outer scope (line 142)

(W0621)